static/app/utils/profiling/profile/eventedProfile.tsx

```diff
@@ -29,6 +31,10 @@ export class EventedProfile extends Profile {
     // If frames are offset, we need to set lastValue to profile start, so that delta between
     // samples is correctly offset by the start value.
     profile.lastValue = Math.max(0, eventedProfile.startValue);
+    profile.samplingIntervalApproximation = makeFormatTo(
+      'milliseconds',
+      eventedProfile.unit
+    )(10);
```

Comment:
nit: there's already a `formatTo` function exported that you can call like `formatTo(10, 'milliseconds', eventedProfile.unit`